torchtext/csrc/vocab.h

```diff
@@ -3,7 +3,9 @@
 namespace torchtext {
 
 typedef std::vector<std::string> StringList;
-typedef c10::Dict<std::string, int64_t> IndexDict;
+typedef ska_ordered::order_preserving_flat_hash_map<std::string, int64_t>
```

Comment:
I'm wondering if we should limit the scope of the PR and only change `IndexDict` here to order flat hash map. Not touching those things related to `get_stoi` func.